examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +46,19 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_welch(epochs, tmin=tmin, tmax=tmax,
+                        fmin=2, fmax=200, picks=picks, n_jobs=1)
+psds = 10 * np.log10(psds)
+psds_mean = psds.mean(0).mean(0)
+psds_std = psds.mean(0).std(0)
+
+ax.plot(freqs, psds_mean, color='k')
+ax.fill_between(freqs, psds_mean - psds_std, psds_mean + psds_std,
+                color='k', alpha=.5)
+ax.set(title='Welch PSD', xlabel='Frequency',
```

Comment:
maybe you could add `(gradiometers)` in the title so that people know it's supposed to look the same as the second subplot.